internal/resources/metal/connection/resource_test.go

```diff
@@ -359,3 +360,44 @@ func TestAccMetalConnection_sharedVlans(t *testing.T) {
 		},
 	})
 }
+
+// Test to verify that switching from SDKv2 to the Framework has not affected provider's behavior
+// TODO (ocobles): once migrated, this test may be removed
+func TestAccMetalConnection_shared_zside_upgradeFromVersion(t *testing.T) {
+	rs := acctest.RandString(10)
+	cfg := testAccMetalConnectionConfig_Shared_zside(rs)
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acceptance.TestAccPreCheckMetal(t) },
+		CheckDestroy: testAccMetalConnectionCheckDestroyed,
+		Steps: []resource.TestStep{
+			{
+				ExternalProviders: map[string]resource.ExternalProvider{
+					"equinix": {
+						VersionConstraint: "1.29.0", // latest version with resource defined on SDKv2
```

Comment:
Does this version number need to be bumped going forward?

Reply:
not necessarily, since it is the latest version available with the resource using sdkv2. In fact, we can remove this test once migrated, it helped validate compatibility